src/main/java/seedu/address/logic/commands/ModFindCommand.java

```diff
@@ -13,14 +13,21 @@
  */
 public class ModFindCommand extends ModCommand {
     public static final String COMMAND_WORD = "find";
-    public static final String MESSAGE_SUCCESS = "Successfully found those who take this mod!";
+    public static final String COMMAND_WORD_TAKEN = "taken";
+    public static final String COMMAND_WORD_TAKING = "taking";
+    public static final String MESSAGE_SUCCESS = "Successfully found those who have this mod!";
```

Comment:
Seems vague, 
"Successfully found those who are taking or have taken this mod!"

Reply:
Okay thanks!